src/test/java/org/ohdsi/authenticator/service/GoogleIapJwtVerifierTest.java

```diff
@@ -0,0 +1,45 @@
+package org.ohdsi.authenticator.service;
+
+import com.nimbusds.jwt.JWTClaimsSet;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Spy;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.ohdsi.authenticator.exception.AuthenticationException;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doReturn;
+import static org.ohdsi.authenticator.service.support.GoogleIapUtils.createJwtToken;
+
+@RunWith(MockitoJUnitRunner.class)
+public class GoogleIapJwtVerifierTest {
```

Comment:
What about REST endpoints tests?

Reply:
do you mean AuthService kind of tests? 
- I don't have a separate AuthService for IAP authorization
- and tbh I don't really like an idea of tests that cannot be run right away, without additional params

Reply:
done